sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -161,9 +162,15 @@ func App() *cli.App {
 					Destination: &useCachedImage,
 					Value:       false,
 				},
+				&cli.StringFlag{
+					Name:        "labels",
+					Usage:       "add labels to the pulled image. Each label should be in the format of `key=value` and separated by a comma",
```

Comment:
What characters are allowed in a label? If either comma or whitespace are allowed then this might not be the right interface.

Reply:
Added here - https://github.com/bottlerocket-os/bottlerocket/pull/3757/files#r1478741201